src/views/List.jsx

```diff
@@ -32,64 +33,74 @@ export function List({ data, listPath }) {
 
 	return (
 		<>
-			{listPath ? <ListHeader text={listName} /> : null}
-			{!listPath ? (
+			{!auth.currentUser ? (
+				<div className="border border-dark-green py-8 px-12 mx-8 md:mx-24 text-off-black text-center tiny:mt-18 xs:mt-32 sm:mt-40 items-center justify-center bg-sage/75 rounded-2xl font-medium">
```

Comment:
future feature: I'll use this element as a basis for the 'about' section i am working on, so the style is coherent for the ui elements.